salt/utils/nxos.py

```diff
@@ -1,4 +1,18 @@
 # -*- coding: utf-8 -*-
+# Copyright (c) 2018 Cisco and/or its affiliates.
```

Comment:
Is this line necessary?

Reply:
This is pretty standard for contributions Cisco makes to open source projects.  Why do you ask?

Reply:
It's not a problem. I'll resolve this review comment.